plugins/jobs/src/js/types/JobRunStatusSummary.ts

```diff
@@ -19,77 +19,14 @@ export function JobRunStatusSummaryTypeResolver(
   history: MetronomeJobHistorySummary
 ): JobRunStatusSummary {
   const summary = JobHistorySummaryTypeResolver(history);
-  let status = "N/A" as JobRunStatus;
-  let time: string | null = null;
-  let lastFailureAt = 0;
-  let lastSuccessAt = 0;
-
-  if (summary.lastFailureAt !== null) {
-    lastFailureAt = DateUtil.strToMs(summary.lastFailureAt);
-  }
-
-  if (summary.lastSuccessAt !== null) {
-    lastSuccessAt = DateUtil.strToMs(summary.lastSuccessAt);
-  }
-
-  if (summary.lastFailureAt !== null || summary.lastSuccessAt !== null) {
-    if (lastFailureAt > lastSuccessAt) {
-      status = "Failed" as JobRunStatus;
-      time = summary.lastFailureAt;
-    } else {
-      status = "Success" as JobRunStatus;
-      time = summary.lastSuccessAt;
-    }
+  if ([summary.lastFailureAt, summary.lastSuccessAt].every(x => x === null)) {
+    return { status: "N/A", time: null };
   }
 
-  return { status, time: time ? DateUtil.strToMs(time) : null };
+  const lastFailureAt = DateUtil.strToMs(summary.lastFailureAt) || 0;
+  const lastSuccessAt = DateUtil.strToMs(summary.lastSuccessAt) || 0;
+  return {
+    status: lastFailureAt > lastSuccessAt ? "Failed" : "Success",
+    time: Math.max(lastFailureAt, lastSuccessAt)
+  };
 }
-
-export const JobRunStatusSummaryFieldResolver = {
```

Comment:
Are we sure that we can remove this complete variable?

Reply:
As it can only be imported by name and the name does not appear anywhere else project-wide (or my grep fools me, which hasn't happened in a long time), i'm quite sure.